platform/core/src/classes/ImageSet.ts

```diff
@@ -10,13 +10,15 @@ const OBJECT = 'object';
  * indiscriminately, but this should be changed).
  */
 class ImageSet {
+  private rawImages;
+
   constructor(images) {
     if (Array.isArray(images) !== true) {
       throw new Error('ImageSet expects an array of images');
     }
 
-    // @property "images"
-    Object.defineProperty(this, 'images', {
+    // @property "rawImages"
```

Comment:
Just set instances to the complete list - instances is used elsewhere to mean all the instance values, whereas images is used for the images for display.